src/resources/views/fields/star.blade.php

```diff
@@ -38,7 +38,7 @@
             }
             .rate:not(:checked) > input {
                 position:absolute;
-                top:-9999px;
+                left:-9999px;
```

Comment:
Hi @siberfx, Thanx for the contribution, if we replace all (`position:absolute; left:-9999px;`) with `display: none;`, will it be more appropriate and cleaner?

Reply:
yes, I already use it this way and its not jumping to top.

Reply:
Okay, waiting for your new commit to merge this pull @siberfx :slightly_smiling_face: